rolling/ci-benchmark.yaml

```diff
@@ -29,8 +29,9 @@ install_packages:
 - ros-noetic-tf2-msgs
 jenkins_job_label: ci-agent
 jenkins_job_priority: 50
-jenkins_job_schedule: 30 23 * * *
```

Comment:
Hm, do we not want to run the rolling jobs nightly either?  Or will this just happen naturally as a result of being a downstream of `nightly-extra-rmw-release`?

Reply:
You're correct. And if that build fails, there's no reason to run this one.

Reply:
Makes sense, thanks for the explanation.